src/lib/images.ts

```diff
@@ -18,22 +16,4 @@ export async function getRemoteImage(src: string) {
     console.error(error);
     return baseBlur;
   }
-}
-
-export async function getLocalImage(staticImport: StaticImport) {
-  try {
-    const src =
```

Comment:
Why get rid of this?

Reply:
There is no need because nextjs does this by default which if I remember you talked about this when you saw it